templates/smart-start/base_smart-start.html

```diff
@@ -0,0 +1,9 @@
+{% extends "templates/base.html" %}
+
+{% block outer_content %}
+  {% block content %}{% endblock %}
+
+  <!-- Set default Marketo information for contact form below-->
+  <div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/smart-start" data-form-id="3591" data-lp-id="" data-return-url="/support/thank-you" data-lp-url="/support/contact-us">
```

Comment:
the data-return-url should be `/smart-start/thank-you`

Reply:
the data-lp-url should be `/smart-start/contact-us`

Reply:
Thanks. Done